src/material-import/material-import.module.ts

```diff
@@ -16,19 +21,29 @@ import {
     MatMenuModule,
     MatButtonModule,
     MatToolbarModule,
-    MatCheckboxModule,
     MatCardModule,
+    FormsModule,
+    ReactiveFormsModule,
```

Comment:
These should not be imported in the Material Import module.  If you want to create a Shared Imports module that would be OK.